### ISSUE TITLE
`jsdoc/require-param` produces syntactically invalid code

### ISSUE THREAD

**Issue by RunDevelopment** [gajus/eslint-plugin-jsdoc#771]
## Expected behavior
When given a single-line doc-comment, the rule should add new lines when adding tags.

E.g.

```js
/** Foo. */
function foo(a, b, c) {}
```

to

```js
/**
 * Foo.
 * @param a
 * @param b
 * @param c
 */
function foo(a, b, c) {}
```

## Actual behavior

```js
* @param a
* @param b
* @param c
/** Foo. */
function foo(a, b, c) {}
```
Note: This happens with both ESLint and the VSCode ESLint extension.

## ESLint Config

```js
/** @type {import("eslint").Linter.Config} */
module.exports = {
	env: {
		browser: true,
		es6: true
	},
	root: true,
	plugins: [
		"jsdoc"
	],
	parserOptions: {
		ecmaVersion: 2018,
		sourceType: "module",
		ecmaFeatures: {
			node: true,
			spread: true
		}
	},
	rules: {
		"jsdoc/require-param": "error"
	},
}

```

## ESLint sample

```js
/** Foo. */
function foo(a, b, c) {}
```

[Example project.](https://github.com/gajus/eslint-plugin-jsdoc/files/6840517/example.zip)

## Environment

- Node version: v14.15.4
- ESLint version v7.26.0
- `eslint-plugin-jsdoc` version: 35.4.1

**Comment by brettz9** [comment]
I changed the behavior to create a normal multiline comment out of a single line comment per your example when missing tags were present.

However, I did not change handling for:

```js
/** up here
*/
```

nor:

```js
/**
down here */
```

...which will currently add the tags below and above the text, respectively.

**Comment by RunDevelopment** [comment]
> ...which will currently add the tags below and above the text, respectively.

So the rule still produces syntactically invalid code?

**Comment by brettz9** [comment]
> > ...which will currently add the tags below and above the text, respectively.
> 
> So the rule still produces syntactically invalid code?

No.

It will produce:

```js
/** up here
 * @param a
 * @param b
 * @param c
 */
```

```js
/**
 * @param a
 * @param b
 * @param c
 * down here */
```

I don't want to get too far into policing where people want things. The latter is semantically invalid, but I'm not planning on adding time and complexity to address every possible bad case.

The single line comment is a reasonable one to support, and that has been fixed.

**Comment by RunDevelopment** [comment]
Thank you for the clarification!
